views/viewEvent.ejs

```diff
@@ -10,6 +10,8 @@
   <link rel="stylesheet" href="/styles/layout.css" type="text/css" />
 
   <script type="text/javascript" src="/vendor/jquery-3.0.0.js"></script>
+  <script type="text/javascript" src="/vendor/bootstrap.bundle.min.js"></script>
+  <script type="text/javascript" src="/vendor/bootstrap.min.js"></script>
   <script type="text/javascript" src="/scripts/app.js"></script>
```

Comment:
Same comment as on createEvent.ejs - are we loading bootstrap with a cdn or hosting - we should discuss and implement the same approach